test/e2e/scenarios/hubble/steps/validate-drop-metric.go

```diff
@@ -0,0 +1,120 @@
+package steps
```

Comment:
Are all steps going to be listed under the same package `steps`? If not should this be under the `hubble` package?

Reply:
Now that everything is here, scenarios and corresponding steps have been moved to respective subdirectories